src/Game/gamePlayReport.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <math/seadVector.h>
+#include "KingSystem/System/PlayReportMgr.h"
+
+namespace uking {
+
+// TODO
+class PlayReport {
```

Comment:
Having a class named PlayReport that isn't the same kind of class as ksys::PlayReport is a bit confusing in my opinion. I'd suggest removing uking::PlayReport, turning the static member functions like reportKorok into non-member functions, and renaming uking::PlayReport::Report into uking::PlayReport (so that uking::PlayReport *is* a ksys::PlayReport).